src/main/java/seedu/duke/commands/ResetBudgetCommand.java

```diff
@@ -0,0 +1,17 @@
+package seedu.duke.commands;
+
+import seedu.duke.data.ShoppingList;
+
+public class ResetBudgetCommand extends Command {
+    public static final String COMMAND_WORD = "RES";
+    public static final String SUCCESS_RESET_BUDGET = "Budget has been reset to $0.00";
+
+    public ResetBudgetCommand() {
+    }
+
+    @Override
+    public CommandResult execute() {
+        ShoppingList.resetBudget();
```

Comment:
Since budget is another object on its own, the reset budget function should be in the Budget class. You may have to pass in the Budget object as an argument for the execute function as well so the command knows which Budget object to invoke the reset budget function for.

Reply:
Thanks!! Will make the changes!!